doc/Events.md

```diff
@@ -10,6 +10,7 @@ to a widget, you can simply invoke the handleEvent method like the following:
 ```lua
 widget_foo:handleEvent(Event:new("Timeout"))
 ```
+However if the widget might be destroyed during the event you can call @{ui.uimanager:sendEvent|UIManager:sendEvent} to propagate the event from the topmost widget or @{ui.uimanager:broadcastEvent|UIManager:broadcastEvent} to send the event to all widgets.
```

Comment:
Technically UIManager, well, *sends* ^^ the event *to* the topmost widget with a matching handler. And, since widgets at that level are usually based on `WidgetContainer`, *that* widget then propagates it to its children.

Reply:
@NiLuJe Something like this?
```suggestion
If the widget can be destroyed during the event you should call @{ui.uimanager:sendEvent|UIManager:sendEvent} to propagate the event from the topmost widget or @{ui.uimanager:broadcastEvent|UIManager:broadcastEvent} to send the event to all widgets with a matching handler.
```

Reply:
Actually, just re-reading it now, I'm fine with the current wording ;p.

Reply:
Alright, I'd still make these changes though. :-)
```suggestion
If the widget can be destroyed during the event you should call @{ui.uimanager:sendEvent|UIManager:sendEvent} to propagate the event from the topmost widget or @{ui.uimanager:broadcastEvent|UIManager:broadcastEvent} to send the event to all widgets.
```